website/source/docs/providers/aws/d/db_snapshot.html.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: "aws"
+page_title: "AWS: aws_db_snapshot"
+sidebar_current: "docs-aws-datasource-db-snapshot"
+description: |-
+  Get information on a DB Snapshot.
+---
+
+# aws\_db\_snapshot
```

Comment:
I'm confused about this data source; it sounds singular but some of the parameters seem like it could return multiple?